kraken/node_actions/openstack_node_scenarios.py

```diff
@@ -0,0 +1,133 @@
+import sys
+import time
+import logging
+import subprocess
+import requests
+import kraken.kubernetes.client as kubecli
+import kraken.invoke.command as runcommand
+import kraken.node_actions.common_node_functions as nodeaction
+from kraken.node_actions.abstract_node_scenarios import abstract_node_scenarios
+
+
+class OPENSTACKCLOUD:
+    def __init__(self):
+        self.Wait = 30
+
+    # Start the node instance
+    def start_instances(self,node):
+        runcommand.invoke("openstack server start %s" % (node))
```

Comment:
Thought about actually using the openstack client instead of subprocessing commands?